### PR TITLE
Fixing APPCLOUD-287

### PR DESCRIPTION
**Pull request by danushkaf**

### CODE REVIEW

File: modules/jaggeryapps/appmgt/src/site/themes/default/templates/envs/template.jag [hunk 1]
```diff
@@ -241,44 +242,56 @@ $(document).ready(function() {
...
-            $(this).parent().parent().parent().find('.element-key-holder')[0].readOnly = true;
-            $(this).parent().parent().parent().find('.element-value-holder')[0].readOnly = true;
-
-            displayAppRedeployNotification();
+            jagg.post("../blocks/application/application.jag", {
+                action: "isEnvironmentVariableExist",
+                versionKey: '<%=versionKey%>',
+                key: updatedKey,
+            }, function (result) {
+                if (result && result.trim() == "true") {
```
Comment (SabraO) [inline modules/jaggeryapps/appmgt/src/site/themes/default/templates/envs/template.jag]: Check and remove the trim

File: modules/jaggeryapps/appmgt/src/site/themes/default/templates/envs/template.jag [hunk 2]
```diff
@@ -241,44 +242,56 @@ $(document).ready(function() {
...
+                    jagg.message({
+                        content: "Environmental variable with key : " + updatedKey + " already exists.",
```
Comment (SabraO) [inline modules/jaggeryapps/appmgt/src/site/themes/default/templates/envs/template.jag]: Change to Environment

File: modules/jaggeryapps/appmgt/src/site/themes/default/templates/envs/template.jag (hunk 1 above)
Comment (danushkaf) [inline modules/jaggeryapps/appmgt/src/site/themes/default/templates/envs/template.jag]: Cant remove it. Without trim does not work. Tested

File: modules/jaggeryapps/appmgt/src/site/themes/default/templates/envs/template.jag (hunk 2 above)
Comment (danushkaf) [inline modules/jaggeryapps/appmgt/src/site/themes/default/templates/envs/template.jag]: Fixed in #728

File: modules/jaggeryapps/appmgt/src/site/themes/default/templates/envs/template.jag (hunk 1 above)
Comment (danushkaf) [inline modules/jaggeryapps/appmgt/src/site/themes/default/templates/envs/template.jag]: Managed it using parse #729

**Pull request merged**